src/main/java/seedu/staffsnap/model/applicant/UniqueApplicantList.java

```diff
@@ -45,7 +51,11 @@ public void add(Applicant toAdd) {
         if (contains(toAdd)) {
             throw new DuplicateApplicantException();
         }
-        internalList.add(toAdd);
+        try {
+            internalList.add(toAdd);
+        } catch (Exception e) {
```

Comment:
A bit of Pokemon Exception Handling over here. Notice that `catch(Exception e)` is never used in the codebase.
I'm unsure if this is the best way to catch the errors here. 

If we must do this for now, perhaps add in a brief comment that this is a short-term fix for v1.2 that will be improved upon later.

Reply:
Yeah it’s highly highly unlikely it’ll be used because it’s only caught when the add in ObservableList<E> throws an error. The most possible error is the null exception, but since that its caught by the contains method above, idk it’ll ever be caught by this method. I’ll write a comment here about the short-term fix. Will probably revert this in later iterations because its really unnecessary, I only wrote this to have something to show for the issue